README.md

```diff
@@ -22,16 +22,31 @@ Fides (*fee-dhez*, Latin: Fidēs) is an open-source tool that allows you to easi
 
 ## :rocket: Quick Start
 
-1. Get running with Docker: First, ensure that you have `make` and `docker` installed locally, and clone the Fides repo. Then, from the fides directory, run the following commands:
+### System Requirements
+1. Docker (20.10.8+)
+2. Python (3.9)
```

Comment:
we actually use python 3.8 but it doesn't really make a difference here, just a note